debian/mariadb-server-10.5.install

```diff
@@ -100,4 +101,5 @@ usr/share/man/man1/wsrep_sst_mariabackup.1
 usr/share/man/man1/wsrep_sst_mysqldump.1
 usr/share/man/man1/wsrep_sst_rsync.1
 usr/share/mysql/errmsg-utf8.txt
+usr/share/mysql/wsrep.cnf
```

Comment:
wsrep.cnf is odd. - 60-galera.cnf covers all the settings

Reply:
Sources build this file, so I think Debian packaging should include it. In general, the Debian packaging at upstream should ship everything and follow the CMake output as closely as possible, and if files are wrong or unnecessary, then remove them on CMake level and not just "mask" the problem by fixing it on packaging level.